datahub-web-react/src/conf/theme/theme_light.config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "styles": {
+        "@border-radius-base": "8px",
+        "@layout-header-background": "#333E4C",
+        "@layout-header-color": "white",
+        "@layout-body-background": "white",
+        "@component-background": "white",
+        "@body-background": "white",
+        "@border-color-base": "#333E4C",
+        "@text-color": "fade(black, 85%)",
+        "@text-color-secondary": "fade(black, 45%)",
+        "@heading-color": "fade(black, 85%)",
+        "@background-color-light": "hsv(0, 0, 98%)",
+        "@divider-color": "fade(black, 6%)",
+        "@disabled-color": "fade(black, 25%)",
+        "@steps-nav-arrow-color": "fade(black, 25%)",
+        "@homepage-background-upper-fade": "#132935",
+        "@homepage-background-lower-fade": "#FFFFFF"
+    },
+    "assets": {
```

Comment:
I like the centralization!! Is there a place we should document what values we support? Maybe in another PR?

Reply:
Oops -- looks like we have it below -- awesome!